Source/loadsave.cpp

```diff
@@ -736,6 +736,11 @@ void LoadMonster(LoadHelper *file, Monster &monster, MonsterConversionData *mons
 
 	if (monster.mode == MonsterMode::Petrified)
 		monster.animInfo.isPetrified = true;
+
+	if (monster.isUnique() && !IsUniqueMonsterValid(monster)) {
+		// if the unique monster is no longer valid (e.g. because the loaded mods changed and the unique monsters changed as a consequence), then this monster will cease to be unique, but will otherwise maintain its properties
+		monster.uniqueType = UniqueMonsterType::None;

```

Comment:
I think i would lean more in the direction of skipping the invalid monster, and we should probably have either an assert or at least a log line here to notify that something unexpected happened in case this starts happening for the wrong reason.